packages/libllvm/build.sh

```diff
@@ -93,6 +96,18 @@ termux_step_post_make_install () {
 	for tool in clang clang++ cc c++ cpp gcc g++ ${TERMUX_HOST_PLATFORM}-{clang,clang++,gcc,g++,cpp}; do
 		ln -f -s clang-${_PKG_MAJOR_VERSION} $tool
 	done
+
+	local OPENMP_ARCH
+	if [ $TERMUX_ARCH = "i686" ] || [ $TERMUX_ARCH = "x86_64" ]; then
+		OPENMP_ARCH="i386"
```

Comment:
I guess you're having clang generate x86 code on x64, so just followed suit here.